packages/web-app-admin-settings/src/components/Users/SideBar/EditPanel.vue

```diff
@@ -161,11 +168,15 @@ export default defineComponent({
         (g) => !selectedGroups.some((s) => s.id === g.id) && !g.groupTypes?.includes('ReadOnly')
       )
     })
-
     const isLoginInputDisabled = computed(() => currentUser.uuid === (props.user as User).id)
+    const readOnlyUserAttributes = useCapabilityReadOnlyUserAttributes()
+    const isInputFieldReadOnly = (key) => {
+      return readOnlyUserAttributes.value.includes(key)
```

Comment:
Quick reminder that we usually prefer `unref()` :)

```suggestion
      return unref(readOnlyUserAttributes).includes(key)
```